_schema.graphql

```diff
@@ -81,6 +81,7 @@ scalar DateTime
 
 type Query {
   collections(
+    category: String
```

Comment:
I was thinking an enum could make sense here but on second thought we don't know what categories are going to look like in the near future.